backend/models/rom.py

```diff
@@ -32,18 +38,48 @@ class Rom(BaseModel):
     name: str = Column(String(length=350))
     slug: str = Column(String(length=400))
     summary: str = Column(Text)
-
-    path_cover_s: str = Column(Text, default=DEFAULT_PATH_COVER_S)
-    path_cover_l: str = Column(Text, default=DEFAULT_PATH_COVER_L)
-    url_cover: str = Column(Text, default=DEFAULT_PATH_COVER_L)
+    total_rating: str = Column(String(length=350))
+    # aggregated_rating: str = Column(String(length=350))
+    genres: JSON = Column(JSON, default=[])
+    # alternative_names: JSON = Column(JSON, default=[])
+    # path_artwork: str = Column(Text, default="")
+    # url_artwork: str = Column(Text, default="")
+    franchises: JSON = Column(JSON, default=[])
+    collections: JSON = Column(JSON, default=[])
+    expansions: JSON = Column(JSON, default=[])
+    # path_cover_s_expansions: str = Column(Text, default="")
+    # path_cover_l_expansions: str = Column(Text, default="")
+    # url_cover_expansions: str = Column(Text, default="")
+    dlcs: JSON = Column(JSON, default=[])
+    # path_cover_s_dlcs: str = Column(Text, default="")
+    # path_cover_l_dlcs: str = Column(Text, default="")
+    # url_cover_dlcs: str = Column(Text, default="")
+    companies: JSON = Column(JSON, default=[])
+    # platforms: JSON = Column(JSON, default=[])
+    first_release_date: int = Column(BigInteger(), default=0)
+    # game_modes: JSON = Column(JSON, default=[])
+    # player_perspectives: JSON = Column(JSON, default=[])
+    # ports: JSON = Column(JSON, default=[])
+    remasters: JSON = Column(JSON, default=[])
+    remakes: JSON = Column(JSON, default=[])
+    # similar_games: JSON = Column(JSON, default=[])
+    # language_supports: JSON = Column(JSON, default=[])
+    # external_games: JSON = Column(JSON, default=[])
+    # external_games_category: JSON = Column(JSON, default=[])
+    expanded_games: JSON = Column(JSON, default=[])
+    # expanded_games_category: JSON = Column(JSON, default=[])
```

Comment:
i have questions about the use of so many loose json fields in this model, especially when it relates to integration data from other sources (like mobi.  i might play around with this is bit before you merge the branch into main.